integration/lsp/test_utils.ts

```diff
@@ -70,11 +71,12 @@ export function initializeServer(client: MessageConnection): Promise<lsp.Initial
   });
 }
 
-export function openTextDocument(client: MessageConnection, filePath: string) {
+export function openTextDocument(
+    client: MessageConnection, filePath: string, languageId = 'typescript') {
```

Comment:
nit: I think it's better we just infer languageId from filePath extension.

Reply:
👍 Yea, that's much better. Thanks for the suggestion